File: API/Entities/Job.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entities
{
    public class Job
    {
        [Required]
        public int Id { get; set; } 

        public string  JobTitle { get; set; } 

        public string Desc { get; set; } 
        

        public string Requirements { get; set; }   

        public string Timeframe { get; set; }  

        public int Budget { get; set; }

        public int FieldId { get; set; }

        public Field Field { get;  set; }

        public int SmeId { get; set; } 

        public Sme Sme { get; set; }

         public ICollection<Bid> Bid { get; set; }
    }
}

File: client/src/app/_models/job.ts
import{ Bid } from '../_models/Bid';
export interface Job{

    id: string;
    jobTitle : string;
    desc :string;
    requirements: string;
    timeframe: string;
    fieldId: string;
    filePath : string;
    budget:string;
    smeId : string;
    compName : string;
    jobStatus : string;
    bid : Bid[];

}